apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/store.js

```diff
@@ -44,6 +51,8 @@ const welcomeGuideVariantReducer = ( state = 'tour', action ) => {
 	switch ( action.type ) {
 		case 'WPCOM_WELCOME_GUIDE_FETCH_STATUS_SUCCESS':
 			return action.response.variant;
+		case 'WPCOM_WELCOME_GUIDE_RESET_STORE':
+			return 'tour';
```

Comment:
I'm a bit curious about this - it's a bit surprising to see this "variant" changed, and I couldn't immediately see what the variant is for.

Reply:
Not sure if you mean why reseting the store sets it back to `'tour'`, in which case it's just the default value for this reducer. If the default was `'modal'` then resetting it should go back to that value.

If you meant what is variant value for, it's either `'tour'` or `'modal'` and the value comes from the backend. The backend decides whether the user should be seeing the welcome guide in the modal (e.g. if they're on a mobile device) or the "tour" which is the floating window that appears in the bottom left.